### PR TITLE
Add info about netCDF support

### PR DESCRIPTION
**Pull request by ninsbl**
Need to add a screenshot or the like.

The Norwegian Ground segment provides sentinel data as NetCDF. So with this addon it should be possible to build a comprehensive STRDS with Sentinel data over Norway, without downloading large amounts of data...
Therefor I put the slide into the t./sentinel section...
Also climate data (that the CF-convention is made for) would be right at your finger tips...

Any suggetions for changes most welcome.

### CODE REVIEW

File: wageningen2021.html [hunk 1]
```diff
@@ -491,6 +491,35 @@ <h2>Importing</h2>
 </section>
 
 
+<section>
+    <h2>Tap directly into FAIR data warehouses</h2>
+    <h4>GRASS GIS understands netCDF data that follows the CF-convention</h4>
+    <div class="container">
+        <div class="col">
+        <ul>
+             <li><b>m.crawl.thredds</b> - crawls data ware houses (thredds server) for netCDF datasets</li>
```
Comment (veroandreo) [inline wageningen2021.html:500]: Can any type of data be stored in netCDF? I'm trying to grasp why using m. for the module name... Also, would you provide a link to this thredds server?

File: wageningen2021.html [hunk 2]
```diff
@@ -491,6 +491,35 @@ <h2>Importing</h2>
...
+             <li><b>t.rast.import.netcdf</b> - Makes Spatio-temporal data in netCDF format directly available for analysis in GRASS STRDS (also without downloading)</li>
+        </ul>
+        </div>
+        <div class="col">
+            Screenshot placeholder
+        </div>
+    </div>
+        <small>
+        Enjoy data delivert right into your GRASS GIS database from e.g.:
```
Comment (veroandreo) [inline wageningen2021.html:509]:
```suggestion
        Enjoy data delivered right into your GRASS GIS database from e.g.:
```

**Comment by veroandreo** [comment]
I reordered the presentation quite a bit now. Where should I include these or at least one of them? I have saved a place in "New addons" but maybe it makes more sense somewhere else?

File: wageningen2021.html [hunk 3]
```diff
@@ -491,6 +491,35 @@ <h2>Importing</h2>
 </section>
 
 
+<section>
+    <h2>Tap directly into FAIR data warehouses</h2>
+    <h4>GRASS GIS understands netCDF data that follows the CF-convention</h4>
```
[the remark this answers is not in the document]
Comment (ninsbl) [reply wageningen2021.html:496]: FAIR is short for Findability, Accessibility, Interoperability, and Reuse as guiding principles in scientific data management:
https://www.go-fair.org/fair-principles/

**Comment by ninsbl** [comment]
> I reordered the presentation quite a bit now. Where should I include these or at least one of them? I have saved a place in "New addons" but maybe it makes more sense somewhere else?

That makes totally sense. It would also fit into the Sentinel processing part, as Sentinel data over Norway are provided as netCDF through the [Norwegian Ground Segment](https://www.satellittdata.no/en). The Thredds server for it is here:
https://nbstds.met.no/thredds/catalog.html

I am currently trying to create a demolocation with an STRDS with (links to) hundreds of Sentinel-2 scenes in one STRDS with band references, that is just a few MB heavy...

However, [Thredds Data Server](https://www.unidata.ucar.edu/software/tds/current/) are most commonly used for all sorts of climate related data (observations, models,...)...

File: wageningen2021.html (hunk 1 above)
Comment (ninsbl) [reply wageningen2021.html]: In principle, also point data can be stored in netCDF, amybe also other vector data, but I am not sure.
However, the point is that `m.crawl.thredds` just returns a list of URLs to datasets on a Thredds server, that fulfill search/filter criteria. So it does not handle an spatial data at all...

**Comment by ninsbl** [comment]
I see that this does not fit in directly after restructuring. Let me know if you prefer the contribution in a different form (or if you need some more background info)...

**Comment by veroandreo** [comment]
I included this content in #19 and added you as co-author, but it seems I put the wrong email address because you don't appear as co-committer  :-(

**Pull request closed without merging**